simulator/src/resim/cmd_set_current_time.rs

```diff
@@ -0,0 +1,35 @@
+use clap::Parser;
+use radix_engine::types::*;
+use radix_engine_interface::{modules::auth::AuthAddresses, time::UtcDateTime};
+
+use crate::resim::*;
+
+/// Set the current time
+#[derive(Parser, Debug)]
+pub struct SetCurrentTime {
+    /// UTC date time in ISO-8601 format, such as '2011-12-03T10:15:30Z'.
```

Comment:
Might be worth adding "to second precision" to this explanation, as it's quite common for people to use `:30.000Z` etc which isn't parsable.

Reply:
Have no problem adding that.